### PR TITLE
Intial search mostly copied from Home

### PR DESCRIPTION
**Pull request by kepet19**

### CODE REVIEW

**Comment by Joergenson** [comment]
sorry misclick

**Comment by Joergenson** [comment]
Maybe wait for pull request 18 to get merged ;)

**Comment by kepet19** [comment]
> Maybe wait for pull request 18 to get merged ;)
Np. 
You can link to the pull_request if you write `#` and then the number `18`
`#18` pr: #18

File: components/styles/search-style.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    color: colors.darkGray,
+  },
+  searchBarInputContainer: {
+    backgroundColor: colors.lightGray,
+    borderRadius: 14,
+  },
+  searchBarContainer: {
+    marginHorizontal: config.deviceWidth * 0.05,
+    backgroundColor: colors.white,
+    color: colors.white,
```
Comment (Joergenson) [inline components/styles/search-style.js:65]:
```suggestion
    color: colors.white,
    borderBottomColor: colors.transparent,
    borderTopColor: colors.transparent,
```

Comment (kepet19) [reply components/styles/search-style.js:65]: it is implemented

File: components/styles/post-style.js [hunk 2]
```diff
@@ -89,21 +89,27 @@ export default StyleSheet.create({
     fontWeight: "bold",
     width: config.deviceWidth * 0.2,
   },
+  whl2: {
```
Comment (Joergenson) [inline components/styles/post-style.js:92]: Move this to search style :-)

File: components/views/search.js [hunk 3]
```diff
@@ -1,13 +1,133 @@
 import React from "react";
-import { Text, View } from "react-native";
-import { Standard } from "components/styles/temp-standard-style";
+import { Text, View, Image, FlatList, TextInput } from "react-native";
+import { Chip, ButtonGroup, SearchBar } from "react-native-elements";
+import GradientText from "../colors/gradient-text";
+import Database from "../database";
+import searchStyle from "../styles/search-style";
+import postStyle from "../styles/post-style";
```
Comment (Joergenson) [inline components/views/search.js:7]: Remove this

**Review by Joergenson — CHANGES_REQUESTED** [review]

**Review by Joergenson — APPROVED** [review]

**Review by Kechr — APPROVED** [review]

**Pull request merged**